build/Makefile

```diff
@@ -413,6 +414,12 @@ build-licenses:
 build-required-src-dist:
 	docker run --rm $(common_mounts) $(build_tag) $(mount_path)/build/build-required-src-dist.sh
 
+# Create docker builder used in creating arm64 images
+# According to the docker documentation, to build the images for ARM64 arch CPU 
+# we would need to use buildx command which uses builder https://docs.docker.com/desktop/multi-arch/
+create-arm64-builder:
+	-docker buildx create --name $(BUILDX_ARM64_BUILDER)
```

Comment:
I noticed this silently fails -- what happens if this step fails? Will the `docker buildx` that uses this builder also fail? Since the image doesn't exist?

Reply:
>I noticed this silently fails

So it fails if builder with this name already exists. And if builder already present then `docker buildx` would work. Making this command optional became important to pass the `make build-images -j 4` command step of CI build process.

Reply:
Aaaah! Got it. let's make a comment to that effect.